visualization/app/codeCharta/util/layoutHelper.ts

```diff
@@ -1,10 +1,17 @@
-import { SquarifiedCodeMapNode } from "./treeMapGenerator"
+import { CodeMapNode, State, BlacklistType, BlacklistItem, Node } from "../codeCharta.model"
+import Rectangle from "./algorithm/rectangle"
+import { CodeMapBuilding } from "../ui/codeMap/rendering/codeMapBuilding"
 import { CodeMapHelper } from "./codeMapHelper"
-import { Node, CodeMapNode, BlacklistItem, BlacklistType, State } from "../codeCharta.model"
 import { Vector3 } from "three"
-import { CodeMapBuilding } from "../ui/codeMap/rendering/codeMapBuilding"
 
-export class TreeMapHelper {
+export interface LayoutNode {
+	data: CodeMapNode
+	value: number
+	rect: Rectangle
+	zOffset: number
+}
```

Comment:
As this is our own generic interface, I would rename some of the properties to be more readable
```
export interface LayoutNode {
	node: CodeMapNode
	areaValue: number
	rectangle: Rectangle
	zOffset: number
}
```